### PR TITLE
Fix documentation style

### LINKED ISSUE #564: Improve code documentation
- [ ] Fix all `@param`s format (ex: `@param {paramType} paramName`)
- [ ] Lowercase param types: `string`, `object`, `number`, `boolean` and `array`
- [ ] Remove all `@method`s and `@name`s
- [ ] Remove documentation for all `constructor()`s
- [ ] Remove all `@param @name @class` from controllers, services, factories, directives and filters

### PR DESCRIPTION
**Pull request by ArsalanSavand**
### Bug fixed
Fixes #564

### CODE REVIEW

**Comment by ArsalanSavand** [comment]
Done.

File: app/account/auth/auth_service.js [hunk 1]
```diff
@@ -1,21 +1,7 @@
...
- * @param $stateParams
- * @param API
- *
- * @returns [Factory]
- */
 function AuthService($state, $rootScope, $cookies, $window, $stateParams, API) {
   /**
-   * @method getAuthenticatedUser
    * @desc Return the currently authenticated user
    *
```
Comment (AmirSavand) [inline app/account/auth/auth_service.js]: Remove this line

Comment (AmirSavand) [inline app/account/auth/auth_service.js]: Remove this line

File: app/account/auth/auth_service.js [hunk 2]
```diff
@@ -47,30 +32,27 @@ function AuthService($state, $rootScope, $cookies, $window, $stateParams, API) {
...
+   * @param {string} token
    */
   function setToken(token) {
     $cookies.put("JWT", token);
   }
 
   /**
-   * @method getToken
    * @desc Return token from localStorage
    *
```
Comment (AmirSavand) [inline app/account/auth/auth_service.js]: Remove this line

File: app/account/auth/auth_service.js [hunk 3]
```diff
@@ -92,7 +73,6 @@ function AuthService($state, $rootScope, $cookies, $window, $stateParams, API) {
   }
 
   /**
-   * @method isAuthenticated
    * @desc Check if the current user is authenticated
    *
```
Comment (AmirSavand) [inline app/account/auth/auth_service.js]: Remove this line

File: app/basement/codekit/codekit_service.js [hunk 4]
```diff
@@ -35,10 +28,9 @@ function Codekit($timeout, $window) {
   }
 
   /**
-   * @name entryStatuses
    * @desc All statuses available for entries.
    *
```
Comment (AmirSavand) [inline app/basement/codekit/codekit_service.js]: Remove line

File: app/basement/codekit/codekit_service.js [hunk 5]
```diff
@@ -55,10 +47,9 @@ function Codekit($timeout, $window) {
   }];
 
   /**
-   * @name entryFormats
    * @desc All formats available for entries.
    *
```
Comment (AmirSavand) [inline app/basement/codekit/codekit_service.js]: Remove line

File: app/basement/codekit/codekit_service.js [hunk 6]
```diff
@@ -92,10 +83,9 @@ function Codekit($timeout, $window) {
   };
 
   /**
-   * @name commentStatuses
    * @desc All statuses available for comments.
    *
```
Comment (AmirSavand) [inline app/basement/codekit/codekit_service.js]: Remove line

File: app/basement/codekit/codekit_service.js [hunk 7]
```diff
@@ -118,10 +108,9 @@ function Codekit($timeout, $window) {
   }];
 
   /**
-   * @name teamRoles
    * @desc All roles for team.
    *
```
Comment (AmirSavand) [inline app/basement/codekit/codekit_service.js]: Remove line

File: app/basement/password/password_service.js [hunk 8]
```diff
@@ -1,20 +1,15 @@
...
 
-/**
- * @class Password
- */
 function Password() {
   return function () {
     /**
-     * @name minLength
      * @type {number}
      *
```
Comment (AmirSavand) [inline app/basement/password/password_service.js]: Remove this line

Comment (AmirSavand) [inline app/basement/password/password_service.js]: Remove this line

File: app/basement/password/password_service.js [hunk 9]
```diff
@@ -1,20 +1,15 @@
...
      * @private
```
Comment (AmirSavand) [inline app/basement/password/password_service.js]: This should be first

File: app/basement/password/password_service.js [hunk 10]
```diff
@@ -24,7 +19,6 @@ function Password() {
     ];
 
     /**
-     * @name strengthColors
      * @type {object}
      *
      * @private
```
Comment (AmirSavand) [inline app/basement/password/password_service.js]: Same as above

File: app/basement/password/password_service.js [hunk 11]
```diff
@@ -1,20 +1,15 @@
...
      */
     var minLength = 6;
 
     /**
-     * @name strengthWords
      * @type {object}
      *
      * @private
```
Comment (AmirSavand) [inline app/basement/password/password_service.js]: Same as above

File: app/basement/password/password_service.js [hunk 12]
```diff
@@ -87,7 +78,6 @@ function Password() {
     };
 
     /**
-     * @method getStrength
      * @desc Get strength word based on strength
      *
```
Comment (AmirSavand) [inline app/basement/password/password_service.js]: Remove line

File: app/basement/password/password_service.js [hunk 13]
```diff
@@ -98,7 +88,6 @@ function Password() {
     };
 
     /**
-     * @method getPercentage
      * @desc Get strength percentage
      *
```
Comment (AmirSavand) [inline app/basement/password/password_service.js]: Remove line

File: app/basement/password/password_service.js [hunk 14]
```diff
@@ -118,10 +106,9 @@ function Password() {
     };
 
     /**
-     * @method isValid
      * @desc Validity of password
      *
```
Comment (AmirSavand) [inline app/basement/password/password_service.js]: Remove line

File: app/comment/comment_service.js [hunk 15]
```diff
@@ -1,55 +1,39 @@
...
 function Comment($rootScope, toaster, API, ModalsService, Codekit) {
   return function (data) {
 
     /**
-     * @name self
      * @desc Super variable for getting this in functions
      *
      * @type {Dolphin}
      */
     var self = this;
```
Comment (AmirSavand) [inline app/comment/comment_service.js]: Only needs `@private`, remove others.

File: app/dolphin/dolphin_service.js [hunk 16]
```diff
@@ -1,41 +1,29 @@
...
      *
      * @type {Dolphin}
      */
     var self = this;
 
     /**
-     * @name data
+     * @readonly
      * @desc Backend data
      *
```
Comment (AmirSavand) [inline app/dolphin/dolphin_service.js]: Remove line

File: app/entry/entry_service.js [hunk 17]
```diff
@@ -1,60 +1,47 @@
...
      *
      * @type {Tag}
      */
     var self = this;
 
     /**
-     * @name data
+     * @readonly
      * @desc Backend data
      *
```
Comment (AmirSavand) [inline app/entry/entry_service.js]: Remove line

File: app/tag/tag_service.js [hunk 18]
```diff
@@ -1,44 +1,31 @@
...
      *
      * @type {Tag}
      */
     var self = this;
 
     /**
-     * @name data
+     * @readonly
      * @desc Backend data
      *
```
Comment (AmirSavand) [inline app/tag/tag_service.js]: Remove line

**Pull request merged**